### DISCUSSION TITLE
LibreTranslate custom instance

### DISCUSSION THREAD

**Discussion by Collide** [minbrowser/min#2066] [Ideas]
At present the API of LibreTranslate in its own instance does not work, it would be good that you can choose other instances or some custom from the configuration panel :).

**Comment by PalmerAL** [comment]
The default instance should be working again, sorry about that!

I think it's unlikely that very many people would have an alternative instance to use, but it's pretty easy to swap out the URL and then recompile: https://github.com/minbrowser/min/blob/master/js/pageTranslations.js#L6

I'm hoping we can switch to [Bergamot](https://github.com/browsermt/bergamot-translator) pretty soon; the translation quality seems a bit better, and it can run entirely locally (so faster, more private, and I don't have to keep paying for a server). They don't support as many languages yet, but they're getting closer: https://github.com/mozilla/firefox-translations-models
